src/components/Board/Output/PhraseShare/PhraseShare.component.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+//import { Link } from 'react-router-dom';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { FormattedMessage } from 'react-intl';
+import CopyIcon from '@material-ui/icons/FilterNone';
+import CloseIcon from '@material-ui/icons/Close';
+import ShareButton from '../SymbolOutput/ShareButton';
+import IconButton from '../../../UI/IconButton';
+import Button from '@material-ui/core/Button';
+import {
+  FacebookShareButton,
+  FacebookIcon,
+  TwitterShareButton,
+  TwitterIcon,
+  EmailShareButton,
+  EmailIcon,
+  WhatsappShareButton,
+  WhatsappIcon,
+  RedditShareButton,
+  RedditIcon
+} from 'react-share';
+import withMobileDialog from '@material-ui/core/withMobileDialog';
+import messages from './PhraseShare.messages';
+
+import './PhraseShare.css';
+
+const PhraseShare = ({
+  label,
+  phrase,
+  intl,
+  open,
+  fullScreen,
+  onShareClick,
+  onShareClose,
+  copyLinkAction,
+  style,
+  hidden
+}) => (
+  <React.Fragment>
+    <ShareButton
+      label={label}
+      color="inherit"
+      onClick={onShareClick}
+      style={style}
+      hidden={hidden}
+    />
+    <Dialog
+      open={open}
+      onClose={onShareClose}
+      fullScreen={fullScreen}
+      className="ShareDialog__container"
+    >
+      <DialogTitle className="ShareDialog__title">
+        <FormattedMessage {...messages.title} />
+
+        <IconButton
+          label={intl.formatMessage(messages.close)}
+          onClick={onShareClose}
+        >
+          <CloseIcon />
+        </IconButton>
+      </DialogTitle>
+      <DialogContent className="ShareDialog__content">
+        <div className="ShareDialog__Subtitle">
+        <FormattedMessage {...messages.subtitle} />
+        </div>
+        <div className="ShareDialog__socialIcons">
+          <Button onClick={copyLinkAction} color="primary">
+            <div className="ShareDialog__socialIcons__copyAction">
+              <div>
+                <CopyIcon />
+              </div>
+              <FormattedMessage {...messages.copyLink} />
+            </div>
+          </Button>
+          <Button>
+            <EmailShareButton
+              subject={phrase}
+              body={phrase}
```

Comment:
email subject should not be the phrase itself, use the share board as an example